### ISSUE TITLE
onnx export problem

### ISSUE THREAD

**Issue by Nioolek** [okankop/Efficient-3DCNNs#15]
Environment:
CUDA 10.0
CUDNN 7
TENSORRT 7
PYTORCH 1.2.0

I tried to convert the 3DCNN model to onnx,but the result by onnx and result by pytorch are different.I tried the backbone of resnet18 and mobilenet.Did anybody meet the same problem?

**Comment by okankop** [comment]
I never tried to cenvert these models to onnx, but in principle it should be possible. When I have time, I would try and let you know.

**Comment by poincarelee** [comment]
@okankop  After two years, still didn't try on exporting to onnx ?  I saw several questions about low accuracy of pretrained models, didn't do any update?
